lib/ValidationError.ts

```diff
@@ -3,6 +3,15 @@ import * as zod from 'zod';
 import { joinPath } from './utils/joinPath';
 import { isNonEmptyArray } from './utils/NonEmptyArray';
 
+const MAX_ISSUES_IN_MESSAGE = 99; // I've got 99 problems but the b$tch ain't one
+const ISSUE_SEPARATOR = '; ';
+const UNION_SEPARATOR = ', or ';
+const PREFIX = 'Validation error';
+const PREFIX_SEPARATOR = ': ';
```

Comment:
dev: define as constants since they are both used in `fromZodIssue` and `fromZodError`. DRY